src/utils/bytes-to-proper-size.ts

```diff
@@ -0,0 +1,15 @@
+export const convertBytesToProperSize = (bytes?: number) => {
+  const kilobyte = 1024
+  const megabyte = kilobyte * 1024
+
+  if (!bytes) {
```

Comment:
I guess this is redundant, you can make bytes required and it will be ts error if you don`t pass a parameter to this function

Reply:
it needs to be optional because this function is inside the IconExtentionWithTitle as well, but there may not be this param
<img width="443" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/79856961/1811d5d8-8706-4eec-a892-ba60b74ec2c3">